adoc/chapters/architecture.adoc

```diff
@@ -1033,10 +1033,9 @@ The meanings of these values are identical to those defined in the {cpp} core
 language.
 
 The complete set of memory orders is not guaranteed to be supported by every
-device, nor across all combinations of devices within a context.  The memory
-orders supported by a specific device and context can be queried using
-functionalities of the [code]#sycl::device# and [code]#sycl::context# classes,
-respectively.
+device, nor across all combinations of devices within a context.  The set of
```

Comment:
I wonder if we should describe this in terms of combinations of devices from the same platform, it's somewhat saying the same thing, but since this is querying static capabilities of the platform rather than the state of a context, this might better reflect that.

Reply:
Addressed in 30fa3952949391f2768cce819e9c18015e6b5f5b.